x-pack/platform/packages/shared/file-upload-common/src/telemetry/register_telemetry.ts

```diff
@@ -0,0 +1,263 @@
+        },
+      },
+      file_size_bytes: {
+        type: 'integer',
+        _meta: {
+          description: 'The bucket in which the file size of the analyzed file falls into.',
+        },
+      },
+      num_lines_analyzed: {
+        type: 'integer',
+        _meta: {
+          description: 'The number of lines that were analyzed in the file.',
+        },
+      },
+      num_messages_analyzed: {
+        type: 'integer',
+        _meta: {
+          description: 'The number of messages that were analyzed in the file.',
+        },
+      },
+      java_timestamp_formats: {
+        type: 'keyword',
+        _meta: {
+          description: 'The Java date formats that were detected in the file. Comma delimited.',
+        },
+      },
+      num_fields_found: {
+        type: 'integer',
+        _meta: {
+          description: 'The number of fields that were found in the file.',
+        },
+      },
+      delimiter: {
+        type: 'keyword',
+        _meta: {
+          description: 'The delimiter used in the file.',
+        },
+      },
+      preview_success: {
+        type: 'boolean',
+        _meta: {
+          description: 'Whether the docs preview was successful.',
+        },
+      },
+      analysis_success: {
+        type: 'boolean',
+        _meta: {
+          description: 'Whether the analysis was successful.',
+        },
+      },
+      overrides_used: {
+        type: 'boolean',
+        _meta: {
+          description: 'Whether overrides were used during analysis.',
+        },
+      },
+      analysis_time_ms: {
+        type: 'integer',
+        _meta: {
+          description: 'The time taken to analyze the file in milliseconds.',
+        },
+      },
+    },
+  });
+
+  analytics.registerEventType({
+    eventType: FILE_UPLOAD_EVENT.FILE_UPLOAD,
+    schema: {
+      upload_session_id: {
+        type: 'keyword',
+        _meta: {
+          description: 'The unique identifier for the upload session.',
+        },
+      },
+      file_id: {
+        type: 'keyword',
+        _meta: {
+          description: 'The unique identifier for the file being uploaded.',
+        },
+      },
+      location: {
+        type: 'keyword',
+        _meta: {
+          description:
+            'The location in the app where the file upload analyze action was triggered from.',
+        },
+      },
+      mapping_clash_new_fields: {
+        type: 'integer',
+        _meta: {
+          description: 'The number of new fields that had a mapping clash when uploading the file.',
+        },
+      },
+      mapping_clash_missing_fields: {
+        type: 'integer',
+        _meta: {
+          description:
+            'The number of missing fields that had a mapping clash when uploading the file.',
+        },
+      },
+      file_size_bytes: {
+        type: 'integer',
+        _meta: {
+          description: 'The bucket in which the file size of the analyzed file falls into.',
```

Comment:
As above, is this correct, in that it relates to a bucket? Or is it that actual number of bytes?